### ISSUE TITLE
Having more then one "<" character in a row on post title causes Twitter to discard text in publiscize.

### ISSUE THREAD

**Issue by KirkwallDay** [Automattic/wp-calypso#55646]
### Steps to Reproduce

1. Create a new post, title it something like "Test < Test 1 << test 2 <<< test 3"
2. Publish this post and use publicize to post to twitter. Make sure the tweet box is the same as the title.
3. The tweet in the resulting post will show "Test < Test 1" and that's it. The post will still embed properly.
Screenshots:
The Tweet Text:
![Screen Shot on 2021-08-22 at 10-57-05](https://user-images.githubusercontent.com/49131029/130363550-0a5a003f-51e1-4e77-8315-11a5d68f31c5.png)

The Tweet:
![Screen Shot on 2021-08-22 at 10-55-32](https://user-images.githubusercontent.com/49131029/130363553-114c99a8-472e-416b-9ad5-b1e22e0f101e.png)

### What I expected to happen

Full tweet text should be visible on twitter.

### What actually happened

The tweet is cut off just before the two << symbols.

### Context

Customer reported here: 4190146-zen

### Operating System

macOS

### OS Version

Catelina

### Browser

Firefox

### Browser Version(s)

Firefox 91

### Simple/Atomic

Simple, Atomic

### Number of Users Impacted

All users

### Reproducibility

Consistent

**Comment by Robertght** [comment]
I think this is Publicize territory. @jeherve is that true?

**Comment by jeherve** [comment]
> I think this is Publicize territory

Yup. It would be interesting to know if this happens with other social networks as well, so we can now if the title is stripped on Twitter's end, or on WordPress.com before it gets sent to all Social Networks.

**Comment by KirkwallDay** [comment]
The user in the ticket said LinkedIn and Facebook were OK. However, I checked and the issue happens on LinkedIn as well, only Facebook is unaffected:
Facebook:
![Screen Shot on 2021-08-26 at 08-14-59](https://user-images.githubusercontent.com/49131029/130979138-83b3cf81-bfbb-4e7d-b5b4-478162207ca0.png)

LinkedIn:
![Screen Shot on 2021-08-26 at 08-15-10](https://user-images.githubusercontent.com/49131029/130979126-08bb4c18-5a88-44da-a3cf-992ed3b639cd.png)
